### PR TITLE
fix: DEV-2930: Fix collapsed texts for TextArea

### PR DESCRIPTION
**Pull request by nicholasrq**
Also fixes DEV-2928, DEV-2929

### CODE REVIEW

File: src/components/SidePanels/DetailsPanel/RegionDetails.tsx [hunk 1]
```diff
@@ -43,6 +43,32 @@ const RegionLabels: FC<{result: any}> = ({ result }) => {
   );
 };
 
+const TextResult: FC<{mainValue: string[]}> = observer(({ mainValue }) => {
+  return (
+    <Text mark >
```
Comment (bmartel) [inline src/components/SidePanels/DetailsPanel/RegionDetails.tsx:48]:
```suggestion
    <Text mark>
```

File: src/components/SidePanels/DetailsPanel/RegionDetails.tsx [hunk 2]
```diff
@@ -43,6 +43,32 @@ const RegionLabels: FC<{result: any}> = ({ result }) => {
...
+      {mainValue.map((value: string, i: number) => (
+        <p key={`${value}-${i}`} data-counter={i + 1}>{value}</p>
+      ))}
+    </Text>
+  );
+});
+
+const CnoicesResult: FC<{mainValue: string[]}> = observer(({ mainValue }) => {
+  return (
+    <Text mark >
```
Comment (bmartel) [inline src/components/SidePanels/DetailsPanel/RegionDetails.tsx:58]:
```suggestion
    <Text mark>
```

**Review by bmartel — CHANGES_REQUESTED** [review]
Formatting, rest looks good.

**Review by bmartel — APPROVED** [review]

File: src/components/SidePanels/DetailsPanel/RegionDetails.tsx [hunk 3]
```diff
@@ -43,6 +43,32 @@ const RegionLabels: FC<{result: any}> = ({ result }) => {
...
+  return (
+    <Text mark>
+      {mainValue.map((value: string, i: number) => (
+        <p key={`${value}-${i}`} data-counter={i + 1}>{value}</p>
+      ))}
+    </Text>
+  );
+});
+
+const CnoicesResult: FC<{mainValue: string[]}> = observer(({ mainValue }) => {
```
Comment (hlomzik) [inline src/components/SidePanels/DetailsPanel/RegionDetails.tsx:56]: typo: ChoicesResult

**Review by hlomzik — CHANGES_REQUESTED** [review]

**Review by hlomzik — APPROVED** [review]

**Pull request merged**